code/datums/components/caltrop.dm

```diff
@@ -59,4 +61,7 @@
 						"<span class='userdanger'>You slide on [A]!</span>")
 
 			cooldown = world.time
-		H.Paralyze(25)
+		if(!is_species(/datum/species/squid))
```

Comment:
`if(!is_species(H, /datum/species/squid))`

Reply:
Actually for clarity you should remove the `!` and swap the code around so `else` handles everything except squids.